app.py

```diff
@@ -112,6 +112,17 @@ def get_medicine(user_id, id):
     result = medicine_schema.dump(med)
     return jsonify(result)
 
+# Delete single medicine
+@app.route('/api/v1/user/<user_id>/medicines/<id>', methods=['DELETE'])
+def delete_medicine(user_id, id):
+    user = User.query.get(user_id)
+    med = Medicine.query.get(id)
+    db.session.delete(med)
+    db.session.commit()
+    all_meds = Medicine.query.all()
```

Comment:
Since we are redirecting the user to get all the medicines from their medicine cabinet, do we still need this variable, all_meds?

Reply:
Oooh good catch - this was leftover from before the redirect happened! I've fixed it.